docs/root/modules/ontology/schema.md

```diff
@@ -20,6 +20,27 @@ In this schema, `squares` represent `Abstract Nodes` and `hexagons` represent `S
 | -------- | --------------- | ------- |
 | UserAccount | Represents a user account on a system or service. This label is used to unify accounts from different sources (e.g., Duo, Okta, Tailscale). | [d3f:UserAccount](https://d3fend.mitre.org/dao/artifact/d3f:UserAccount/) |
```

Comment:
Listing out all the available semantic labels is great. Would it make sense to also add in a section that lists out all of the available `_ont_*` fields authoritatively?